dist/src/main/config/gateway.yaml.template

```diff
@@ -231,6 +231,18 @@
       # This setting can also be overridden using the environment variable ZEEBE_GATEWAY_LONGPOLLING_ENABLED.
       # enabled: true
 
+      # Set the timeout for long polling
+      # This setting can also be overridden using the environment variable ZEEBE_GATEWAY_LONGPOLLING_TIMEOUT.
+      # timeout: 10000
+
+      # Set the probe timeout for long polling
+      # This setting can also be overridden using the environment variable ZEEBE_GATEWAY_LONGPOLLING_PROBETIMEOUT.
+      # probeTimeout: 10000
+
+      # Set the number of minimum empty responses
```

Comment:
:wrench: Explain a bit more what that means. For example, just copy a  code comment from `LongPollingActivateJobsHandler`:
> Minimum number of responses with jobCount 0 to infer that no jobs are available